src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -301,57 +271,68 @@ HRESULT STDMETHODCALLTYPE CorProfiler::AssemblyLoadFinished(AssemblyID assembly_
 
   const auto assembly_info = GetAssemblyInfo(this->info_, assembly_id);
   if (!assembly_info.IsValid()) {
+    Debug("AssemblyLoadFinished: ", assembly_id, " ", hr_status);
     return S_OK;
   }
 
-  ComPtr<IUnknown> metadata_interfaces;
-  auto hr = this->info_->GetModuleMetaData(assembly_info.manifest_module_id, ofRead | ofWrite,
-                                           IID_IMetaDataImport2,
-                                           metadata_interfaces.GetAddressOf());
+  const auto is_managed_profiler = assembly_info.name == WStr("Datadog.Trace.ClrProfiler.Managed");
 
-  if (FAILED(hr)) {
-    Warn("AssemblyLoadFinished failed to get metadata interface for module id ", assembly_info.manifest_module_id,
-         " from assembly ", assembly_info.name);
-    return S_OK;
-  }
-
-  // Get the IMetaDataAssemblyImport interface to get metadata from the managed assembly
-  const auto assembly_import = metadata_interfaces.As<IMetaDataAssemblyImport>(
-      IID_IMetaDataAssemblyImport);
-  const auto assembly_metadata = GetAssemblyImportMetadata(assembly_import);
+  if (is_managed_profiler || debug_logging_enabled) {
+    if (debug_logging_enabled) {
+      Debug("AssemblyLoadFinished: ", assembly_id, " ", hr_status);
+    }
 
-  if (debug_logging_enabled) {
-    Debug("AssemblyLoadFinished: AssemblyName=", assembly_info.name, " AssemblyVersion=", assembly_metadata.version.str());
-  }
+    ComPtr<IUnknown> metadata_interfaces;
+    auto hr = this->info_->GetModuleMetaData(assembly_info.manifest_module_id, ofRead | ofWrite, IID_IMetaDataImport2, metadata_interfaces.GetAddressOf());
 
-  if (assembly_info.name == WStr("Datadog.Trace.ClrProfiler.Managed")) {
-    // Configure a version string to compare with the profiler version
-    std::stringstream ss;
-    ss << assembly_metadata.version.major << '.'
-       << assembly_metadata.version.minor << '.'
-       << assembly_metadata.version.build;
+    if (FAILED(hr)) {
+      Warn("AssemblyLoadFinished failed to get metadata interface for module "
```

Comment:
Weird how your auto-formatting removed the line breaks in line 286 above, but split `"id "` into a separate string here.

Reply:
I think is the visual studio plugin to remove the blank spaces messing with the google format for c++ or something.